src/utils/searchViewHelper.js

```diff
@@ -31,9 +32,11 @@ const searchViewHelper = {
     // If repository is selected after search value is input, fetch search results
     if (search !== '') {
       searchViewHelper.searchByRevisionOrEmail(repository, search);
+    } else {
+      // Fetch 10 most recent revisions when repository changes
+      store.dispatch(fetchRecentRevisions(repository));
```

Comment:
only fetch recent revisions if there is no search value

Reply:
With this it is much easier to test the code locally 👍